### PR TITLE
fix: executor point to pea again

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

**Review by florian-hoenicke — DISMISSED** [review]

File: jina/parsers/__init__.py [hunk 1]
```diff
@@ -205,13 +205,23 @@ def get_main_parser():
...
             'executor',
-            aliases=['pod'],
+            aliases=['pea'],
             help='Start an Executor',
             description='Start an Executor. Executor is how Jina processes Document.',
             formatter_class=_chf,
         )
     )
 
+    set_pod_parser(
```
Comment (hanxiao) [inline jina/parsers/__init__.py:215]: Looks wrong, shouldn't be set_pea_parser?

File: jina/parsers/__init__.py [hunk 2]
```diff
@@ -205,13 +205,23 @@ def get_main_parser():
     set_pod_parser(
         sp.add_parser(
             'executor',
-            aliases=['pod'],
+            aliases=['pea'],
```
Comment (hanxiao) [inline jina/parsers/__init__.py:208]: Breaking change? Why pod parser is related to pea cli now? Then what is jina pea cli then?

**Review by hanxiao — CHANGES_REQUESTED** [review]

File: jina/parsers/__init__.py (hunk 2 above)
Comment (JoanFM) [reply jina/parsers/__init__.py:208]: will revert

Comment (JoanFM) [inline jina/parsers/__init__.py:208]:
```suggestion
            aliases=['pod'],
```

File: jina/parsers/__init__.py (hunk 1 above)
Comment (JoanFM) [inline jina/parsers/__init__.py:215]:
```suggestion
    set_pea_parser(
```

File: tests/integration/issues/github_3124/test_cli_executor.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+
+def test_zed_runtime_cli_docker(docker_image):
+    process = subprocess.Popen(
+        ['jina', 'zed_runtime', '--uses', 'docker://clitest:latest']
+    )
+    time.sleep(5)
+    poll = process.poll()
+    process.terminate()
+    assert poll == 1  # failed
```
Comment (cristianmtr) [inline tests/integration/issues/github_3124/test_cli_executor.py:36]: But why should this fail if you added https://github.com/jina-ai/jina/pull/3144/files#diff-4f725f5fcab143dccca4a394a853b6a44040931ed04346c06e4e3ac041b979c9R215 ?

File: cli/api.py [hunk 4]
```diff
@@ -47,8 +47,17 @@ def zed_runtime(args: 'Namespace'):
...
-# alias
-executor = zed_runtime
+def executor(args: 'Namespace'):
+    """
+    Starts a ZEDRuntime
+
+    :param args: arguments coming from the CLI.
+    """
+    uses = args.uses
+    if uses.startswith('jinahub+docker://') or uses.startswith('docker://'):
```
Comment (hanxiao) [inline cli/api.py:57]: perhaps also `jinahub://`?

**Review by cristianmtr — APPROVED** [review]

**Review by hanxiao — APPROVED** [review]

**Pull request merged**